### PR TITLE
Telegram

### PR DESCRIPTION
**Pull request by artbird309**
#### Database Migration

 NO
#### Description

Goal is to add Telegram notification in #1355.
#### Todos
- [x] Tested by having it notify my Telegram group from Windows 10 debug build.
- [ ] Documentation
#### Issues Fixed or Closed by this PR
#1355
-

### CODE REVIEW

File: src/NzbDrone.Core/NzbDrone.Core.csproj [hunk 1]
```diff
@@ -1127,6 +1131,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="App.config" />
+    <None Include="ClassDiagram1.cd" />
```
Comment (artbird309) [inline src/NzbDrone.Core/NzbDrone.Core.csproj]: This line should not be included, it was a class diagram I temporarily created and I thought I deleted it all.

**Comment by wjbridge** [comment]
Is there an estimate when this can get pulled? This is a really useful feature :)

**Comment by daystar1124** [comment]
I am also curious when this can get pulled in! I use telegram already and adding in with notify option as well would be fantastic!

**Comment by jcamachocds** [comment]
This would be great!

**Comment by markus101** [comment]
Thanks for getting this together and waiting. I made a couple minor change to the formatting and improved the messaging when testing and merged it in.

**Pull request closed without merging**